executor_manager/services/sandbox/repository.py

```
@@ -0,0 +1,496 @@
+                base_url=base_url,
+                created_at=sandbox_info["created_at"],
+                started_at=sandbox_info.get("started_at"),
+                last_activity_at=sandbox_info.get(
+                    "last_activity_at", sandbox_info["created_at"]
+                ),
+                expires_at=sandbox_info.get("expires_at"),
+                error_message=sandbox_info.get("error_message"),
+                metadata=sandbox_info.get("metadata", {}),
+            )
+
+            return sandbox
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)